### ISSUE TITLE
Bug: Correct processing of MWV message with "True" wind direction reference

### ISSUE THREAD

**Issue by rgleason** [rgleason/watchdog_pi#99]
Fixed with bdb  PR. https://github.com/rgleason/watchdog_pi/commit/27672afd9e8986d50ee769ed1121e0b212b5a340

**Comment by rgleason** [comment]
Now we are getting data, but there are a couple of things that aren't working right.
![Screenshot (1953)](https://github.com/rgleason/watchdog_pi/assets/2363959/a6c079a5-159e-4fa7-a4f0-e70715dfa32f)

![Wind-T-A-True-Absolute](https://github.com/rgleason/watchdog_pi/assets/2363959/890aa5b7-4ff3-408b-8cdf-34171f69f1bd)

VDR file used to test this is Hakan's:  [VDR_Sensors-Hakan.txt](https://github.com/rgleason/watchdog_pi/files/14077393/VDR_Sensors-Hakan.txt)

with Dashboard and the Imported Plugin for Windows Ov5.8.4
[watchdog_pi-2.4.106.0-msvc-x86_64-wx32-10.0.22621-MSVC.tar.gz](https://github.com/rgleason/watchdog_pi/files/14077397/watchdog_pi-2.4.106.0-msvc-x86_64-wx32-10.0.22621-MSVC.tar.gz)

**Comment by rgleason** [comment]
Dave, I got this totally wrong.  
True Wind  is the Direction from which the wind is blowing.  Sorry.
Ignore the 180 in my screenshot.

**Comment by rgleason** [comment]
Closing   Bug: Issue on version 2.4.106 (Wind T-A Alarm) -fixed? #74

**Comment by bdbcat** [comment]
Please try again, simple words.

**Comment by rgleason** [comment]
Yes, you are right Dave it should become the value in Dashboard's "True Wind Direction", when Sync is clicked, but nothing happens sometimes, other times when there is a value in True Wind Direction, it puts something else in there.

Also I notice that sometimes if there is nothing in True Wind Direction, and Sync is clicked it calculates a direction based upon Wind Apparent Relative and SOG or STW?   Do you think that the value calculated is correct?

I also find that the "Sync" button does not always work, it is locked out and nothing will change in the Value field. I don't know what causes this.

Hear are a couple of screenshots using 
[VDR_Sensors-Hakan.txt](https://github.com/rgleason/watchdog_pi/files/14078606/VDR_Sensors-Hakan.txt)


![Wind-T-A-Corrected](https://github.com/rgleason/watchdog_pi/assets/2363959/4c0fbfd7-0bf3-40c6-89af-ca2bfe89c372)
![Screenshot (1965)](https://github.com/rgleason/watchdog_pi/assets/2363959/efa15ff4-ad75-4d53-908e-cfe7300891a9)

**Comment by bdbcat** [comment]
Sync button obviously broken.
I'll fix that first.

**Comment by rgleason** [comment]
Dave, I am attaching a Thomas Rauch  file from the adriatic, which has MWV and is moving. It is an actual recording of Thomas vessel.
[Tactics-sample2-5m.txt](https://github.com/rgleason/watchdog_pi/files/14083921/Tactics-sample2-5m.txt)

![Screenshot (1966)](https://github.com/rgleason/watchdog_pi/assets/2363959/a1c6f46a-19fd-4835-960e-5527958aa22d)

**Comment by rgleason** [comment]
1. Synch button is not working.
2. When the "Value" is entered or selected as say "100" and the Range is "15"  what displays is
85 <  129  < 115
100-15=85    Non-sesnical  100+15= 115
Don't know what creates the middle number.

**Comment by bdbcat** [comment]
PR posted to correct errors.

**Comment by rgleason** [comment]
https://github.com/rgleason/watchdog_pi/pull/105
